openmrs/src/main/java/org/motechproject/openmrs/tasks/constants/Keys.java

```diff
@@ -55,6 +55,11 @@
     public static final String STATE_START_DATE = "startDate";
     public static final String PROGRAM_ENROLLMENT_ATTRIBUTES = "programEnrollmentAttributes";
 
+    //Cohortquery triggers
+    public static final String PATIENT_ID = "PatientId";
+    public static final String PATIENT_DISPLAY = "PatientDisplay";
+    public static final String COHORT_QUERY_ID = "CohortQueryId";
```

Comment:
The user should see "CohortQueryUUID" to keep in line with OpenMRS Reporting REST API definition